cms/articles/tests/test_models.py

```diff
@@ -233,26 +234,203 @@ def test_cite_this_page_is_not_shown_when_unticked(self):
         response = self.client.get(self.basic_page_url)
         self.assertNotContains(response, expected)
 
-    def test_breadcrumb_doesnt_containt_series_url(self):
+    def test_breadcrumb_is_shown(self):
+        # Statistical Articles are a special case regarding breadcrumbs the url includes a series page which is not
+        # accessible externally
         response = self.client.get(self.basic_page_url)
-        # confirm that current breadcrumb is there
-        article_series = self.basic_page.get_parent()
-        self.assertNotContains(
-            response,
-            f'<a class="ons-breadcrumbs__link" href="{article_series.url}">{article_series.title}</a>',
-            html=True,
+        mock_url = response.request["PATH_INFO"]
+        mock_url = mock_url.split("/")
+        mock_url = list(filter("".__ne__, mock_url))
+        mock_breadcrumb = self.get_breadcrumbs(response)
+        expected = 'class="ons-breadcrumbs__link"'
+        self.assertContains(response, expected)
+        self.assertEqual(len(mock_url) - len(mock_breadcrumb), 2)
+        self.assertEqual(mock_url[0], mock_breadcrumb[0])
+        self.assertEqual(mock_url[1], mock_breadcrumb[1])
+
+    def get_breadcrumbs(self, response):
+        mock_response = response.content.decode("utf-8").split("\n")
+        mock_hrf = ""
+        for ind, line in enumerate(mock_response):
+            if "breadcrumb" in line:
+                mock_hrf = mock_response[ind + 1].split("/", 1)
+        mock_breadcrumb = mock_hrf[1][:-1]
+        mock_breadcrumb = mock_breadcrumb.split("/")
+        mock_breadcrumb = list(filter("".__ne__, mock_breadcrumb))
+        return mock_breadcrumb
```

Comment:
question: can you expand why this is needed?

`assertContains`/ `assertNotContains` check that the provided text is present/not present in the response. using `html=True` allows you to search for HTML snippets in a more reliable and efficient than breaking down the content into lines and parsing each on individually